app/views/smart_answers/inputs/_money_question.html.erb

```diff
@@ -4,7 +4,7 @@
   },
   name: "response",
   id: "response",
-  heading_size: "m",
+  heading_size: "xl",
   width: 10,
   hint: question.hint_text,
   value: prefill_value_for(question),
```

Comment:
How come we don't add the smart answer heading as caption for these question?

Reply:
and the other question types below

Reply:
I think I missed it - the component doesn't support it so will need to add it in manually